jm-kedro-project/src/jm_kedro_project/cmd_test.py

```
@@ -0,0 +1,3 @@
+import os
+print("cmd" in os.environ["PATH"])
```

Ruff (0.11.9)

`print` found

Remove `print`

(T201)